### ISSUE TITLE
Show native mouse cursor instead of android cursor

### ISSUE THREAD

**Issue by renhiyama** [waydroid/waydroid#930]
**Is your feature request related to a problem? Please describe.**
I don't feel waydroid is integrated with my desktop because my mouse cursor changes from native gtk based cursor to android one...
**Describe the solution you'd like**
Waydroid should use native mouse cursor icons
**Describe alternatives you've considered**
Didnt...

**Comment by feois** [comment]
Agreed. I can't easily spot where my cursor is when playing games so I set my cursor color as white but waydroid doesn't follow native cursor
![Screenshot_20230601_152543](https://github.com/waydroid/waydroid/assets/68548170/555846dd-8202-456a-a88c-db8f565b1efd)

**Comment by Quackdoc** [comment]
> I don't feel waydroid is integrated with my desktop because my mouse cursor changes from native gtk based cursor to android one...

I believe waydroid uses the cursor when running native, are you running nested?

**Comment by Noob-linux-user** [comment]
same way droid is working great, except it looks way out of place. the border looks ugly, and the window seems to take the whole screen as the overview in gnome looks ugly

**Comment by Only1337** [comment]
I agree, using my native mouse pointer would be more pleasant and harmonious.

**Comment by lisai9093** [comment]
Agree as well. Use cursor for touch operation is much easier and intuitive. Especially when I need to change the android resolution, the cursor simply becomes too large and too quick.

**Comment by randomoOCoder** [comment]
It's been 2 years, is there any news on this

**Comment by Prak005** [comment]
I guess will have to just use the ugly big cursor

**Comment by burtvadr** [comment]
yeah. this is slightly annoying, i hate this ugly big cursor
